src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -280,7 +280,7 @@ def _list_links_ids(self, source: TDRSourceRef, prefix: str) -> List[TDRBundleFQ
         validate_uuid_prefix(prefix)
         current_bundles = self._query_latest_version(source.spec, f'''
             SELECT links_id, version
-            FROM {self._full_table_name(source.spec, 'links')}
+            FROM `{self._full_table_name(source.spec, 'links')}`
```

Comment:
```suggestion
            FROM {backquote(self._full_table_name(source.spec, 'links'))}
```

and `def backquote(s)` at the top of the module.